core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-category-list/dot-category-field-category-list.component.html

```diff
@@ -11,7 +11,10 @@
             class="flex align-content-center align-items-center category-list__item"
             [ngClass]="{ 'category-list__item--selected': item.checked }"
             (click)="itemClicked.emit({ index, item })">
-            <p-checkbox [(ngModel)]="itemsSelected" [value]="item.key" />
+            <p-checkbox
+                [(ngModel)]="itemsSelected"
```

Comment:
@oidacra To avoid this part in TS 

```
readonly #effectRef = effect(() => {
        this.itemsSelected = this.selected();
    });
```

Is it possible to send a signal directly in an ngModel like this:  `<input type="text" [(ngModel)]="$selected" />`

Here a [ref](https://stackblitz.com/edit/stackblitz-starters-h9s64d?file=src%2Fmain.ts)

Reply:
I will leave as is. I will change it in the next PR. Thanks.

Reply:
TIL! Nice @nicobytes